server/server.go

```diff
@@ -773,7 +786,7 @@ func (s *Server) waitForDrain() {
 }
 
 func (s *Server) LogStreaming(w http.ResponseWriter, _ *http.Request) {
```

Comment:
You can delete this function completely.

Reply:
👍